libs/barista-components/button-group/src/button-group.ts

```diff
@@ -194,6 +194,11 @@ export class DtButtonGroupItem<T> extends _DtButtonGroupItem
     HasTabIndex,
     AfterContentInit,
     OnDestroy {
+  /**
+   * @deprecated
+   * @breaking-change Will be renamed from 'selected' to 'checked' with v 9.0.0.
+   */
+  /* tslint:disable:no-unused-variable */
```

Comment:
the deprecation has to be on the `@Inpu`t there you deprecate a private property a private does not have to be deprecated.